static/module/list.js

```diff
@@ -116,32 +132,47 @@ export class List {
 
     const queryString = params.toString()
     const queryString_ = queryString ? '?' + queryString : ''
-
-    // Only push state if URL is actually changing
-    if (window.location.search !== queryString_) {
-      const target = '/' + queryString_
-      const title = value.length > 0 ? `Search — ${value}` : this.initialTitle
+    const target = queryString_ ? '/' + queryString_ : this.initialPath
+    const currentPath = `${window.location.pathname}${window.location.search}`
+    const sortTitle = this.sortTitleMap[sortBy] ?? sortBy
+    const title
+      = usingWildcard
+        ? `List by ${sortTitle}`
```

Comment:
Teensy tiny minor point: currently this becomes a kind of title case ("List by Newest"), where I think it might look more natural if the sortTitle's first character is lowercase ("List by newest").

Other than that: quite nice 👌🏻

Reply:
It is called title case for a reason.  `const title = ...`

Reply:
Fair enough 😄 I don't have a strong opinion, if you prefer it as is let's go with that.